src/helpers/search.ts

```diff
@@ -84,3 +86,22 @@ export function isSearchHistoryEntity(obj: unknown): obj is SearchHistory {
 		(obj as SearchHistory).results !== undefined
 	);
 }
+
+export function getEquilizedFilterState(filter: FilterState) {
+	return Object.fromEntries(
+		Object.entries(filter)
+			.filter(
+				([key, value]) => key !== 'status' && value && value.values && value.values.length > 0,
+			)
+			.map(([key, value]) => [
+				key,
+				toJS(
+					value
+						? typeof value.values === 'string'
+							? value
+							: { ...value, values: toJS(value.values).sort() }
```

Comment:
why do we need to sort it ?

isnt toJS recursive? i dont think we need this inner toJS call